### ISSUE TITLE
[Bug] Settings menu is broken 

### ISSUE THREAD

**Issue by Austin12325** [RandyKnapp/Auga#119]
Newest Valheim update breaks the UI, biggest issue is the back button quits working and a restart is required. 

![image](https://user-images.githubusercontent.com/23369696/187573533-8d451e48-9b1c-4da9-84a2-312790347a77.png)

**Comment by rmdwirizki** [comment]
To add more context, this is the log output after each action:

**When Opening the Setting**
```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Settings.LoadSettings () (at <9be67623c6d74705a14f96d3a8ff485d>:0)
(wrapper dynamic-method) Settings.DMD<Settings::Awake>(Settings)
UnityEngine.Object:Instantiate(GameObject, Transform)
FejdStartup:OnButtonSettings()
UnityEngine.EventSystems.EventSystem:Update()
```

**When clicking Back**
```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Settings.LoadSettings () (at <9be67623c6d74705a14f96d3a8ff485d>:0)
Settings.OnBack () (at <9be67623c6d74705a14f96d3a8ff485d>:0)
UnityEngine.Events.InvokableCall.Invoke () (at <c9fb31c2eee14ce4aeb36e4c61e154d8>:0)
UnityEngine.Events.UnityEvent.Invoke () (at <c9fb31c2eee14ce4aeb36e4c61e154d8>:0)
UnityEngine.UI.Button.Press () (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.UI.Button.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.EventSystems.EventSystem:Update()
```

**When clicking Apply**
```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Settings.SaveSettings () (at <9be67623c6d74705a14f96d3a8ff485d>:0)
Settings.OnOk () (at <9be67623c6d74705a14f96d3a8ff485d>:0)
UnityEngine.Events.InvokableCall.Invoke () (at <c9fb31c2eee14ce4aeb36e4c61e154d8>:0)
UnityEngine.Events.UnityEvent.Invoke () (at <c9fb31c2eee14ce4aeb36e4c61e154d8>:0)
UnityEngine.UI.Button.Press () (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.UI.Button.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <67859b8eac9e4f3b99606821e32b26c6>:0)
UnityEngine.EventSystems.EventSystem:Update()
```

**Comment by RandyKnapp** [comment]
Should be fixed as of 1.0.12
